### ISSUE TITLE
pure Go MTP library does not work with Samsung non-nexus devices 

### ISSUE THREAD

**Issue by hanwen** [hanwen/go-mtpfs#29]
TL;DR if you have problems connecting a Samsung device (non-nexus), try the libmtp branch rather than master. 

This bug is to document all Samsung issues together.

Several users have reported issues with the master branch and samsung galaxy devices (eg. Galaxy 3, Galaxy 2 etc.). Samsung is using their own MTP stack, which does not quite work like the standard Android one, and I don't have a Samsung device to debug things. Users with an affected device can try the 'libmtp' branch which is an older version linking against libmtp.

Symptoms are:
- Configure failed: OpenSession after reset: LIBUSB_ERROR_TIMEOUT

no matter what the setting of -usb-timeout.
